### PR TITLE
convert to vue step1

### PR DESCRIPTION
**Pull request by snowwhitedev**
I started converting current project to vue front.  I configured all environment and added one new django app frontend. But please pay attention config/setting.py and url.py. I made minor changes there.
Thanks.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -130,3 +130,6 @@ node_modules
 docker-compose.override.yml
 
 *.code-workspace
+
+#copy of original app
+app_old
```
Comment (kennethjiang) [inline .gitignore:135]: please remove

File: docker-compose.yml [hunk 2]
```diff
@@ -54,6 +54,7 @@ services:
     depends_on:
       - ml_api
     command: sh -c "python manage.py collectstatic --noinput && python manage.py migrate && python manage.py runserver --noreload 0.0.0.0:3334"
+    # command: sh -c "python manage.py collectstatic --noinput && python manage.py migrate && python manage.py runserver 0.0.0.0:3334"
```
Comment (kennethjiang) [inline docker-compose.yml:57]: please remove

File: web/app/templates/layout.html [hunk 3]
```diff
@@ -67,7 +67,7 @@
         <ul class="navbar-nav">
           {% if user.is_authenticated %}
           <li class="nav-item {% if  'printers' in view_name %}active{% endif %}">
-            <a class="nav-link" href="/printers/">Printer
+            <a class="nav-link" href="/printers/">Printer AASS
```
Comment (kennethjiang) [inline web/app/templates/layout.html:70]: please revert this change

File: web/app/views.py [hunk 4]
```diff
@@ -21,11 +21,14 @@
...
+#         return redirect('/consent/')
+#     else:
+#         return redirect('/printers/')
+
 def index(request):
-    if request.user.is_authenticated and not request.user.consented_at:
-        return redirect('/consent/')
-    else:
-        return redirect('/printers/')
+    return render(request, 'frontend/index.html')
```
Comment (kennethjiang) [inline web/app/views.py:31]: this broke the app. please revert

File: web/app/views.py [hunk 5]
```diff
@@ -153,14 +156,18 @@ def unsubscribe_email(request):
...
-    prints = get_prints(request).filter(video_url__isnull=False).order_by('-id')
-    if request.GET.get('deleted', False):
-        prints = prints.all(force_visibility=True)
-    page_obj = get_paginator(prints, request, 9)
-    prediction_urls = [ dict(print_id=print.id, prediction_json_url=print.prediction_json_url) for print in page_obj.object_list]
-    return render(request, 'print_list.html', dict(prints=page_obj.object_list, page_obj=page_obj, prediction_urls=prediction_urls))
+    print("prints page")
+    return render(request, 'frontend/index.html')
 
 @login_required
```
Comment (kennethjiang) [inline web/app/views.py:172]: this broke the app. please revert

File: web/config/settings.py [hunk 6]
```diff
@@ -31,6 +31,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get('DEBUG') == 'True'
 
+DEBUG = True
```
Comment (kennethjiang) [inline web/config/settings.py:34]: this broke the app. please revert

File: .gitignore (hunk 1 above)
Comment (snowwhitedev) [reply .gitignore:135]: I will remain while developing.

File: docker-compose.yml (hunk 2 above)
Comment (snowwhitedev) [reply docker-compose.yml:57]: Removed.

File: web/app/templates/layout.html (hunk 3 above)
Comment (snowwhitedev) [reply web/app/templates/layout.html:70]: removed

File: web/app/views.py (hunk 4 above)
Comment (snowwhitedev) [reply web/app/views.py:31]: reverted. I changed it to check merging Vue front to app.

File: web/app/views.py (hunk 5 above)
Comment (snowwhitedev) [reply web/app/views.py:172]: I will change it after doing all components in prints page. 
This is not a bug nor a mistake.

File: web/config/settings.py (hunk 6 above)
Comment (snowwhitedev) [reply web/config/settings.py:34]: This is required for development stage. It will be changed after developing.

Comment (kennethjiang) [reply web/config/settings.py:34]: Please set `DEBUG` env var  to True, rather than hard-coding it.
